langlib/jballerina.java/src/main/ballerina/annotations.bal

```diff
@@ -126,3 +126,12 @@ public const annotation FieldData FieldSet on source external;
 # }
 # ```
 public const annotation ObjectData Binding on class;
+
+# Identifies a type, class, or function that is used in the Java native code. This annotation should be added by the
+# developer if the type, class, or function could be accessed via an `io.ballerina.runtime.api.creators` API such as
```

Comment:
```suggestion
# developer if the type, class, or function could be accessed via runtime APIs such as `ValueCreator`
```
WDYT?

Reply:
I think its better. Refactored in 6d3dd2d12312dc2613b3dc83b67433e633510f18.